Realm/Swift/RLMSupport.swift

```diff
@@ -19,32 +19,55 @@
 import Realm
 
 extension RLMRealm {
+    /**
+     Swift wrapper for `+ [RLMRealm schemaVersionAtURL:`.
```

Comment:
Bad formatting.

Reply:
Which is better?
```
    /**
     Returns the schema version for a Realm at a given local URL.
     
     - see: `RLMRealm.schemaVersionAtURL`
     */
```
or 
```
    /**
     Returns the schema version for a Realm at a given local URL.
     
     - see: `- [RLMRealm schemaVersionAtURL:encryptionKey:error:]`
     */
```

Reply:
We have a couple cases where we reference swift methods in comments, but not Objc. We only use `- see` for RLM classes. So I'm kinda guessing.